src/ImageRunModal.jsx

```diff
@@ -351,68 +353,116 @@ export class ImageRunModal extends React.Component {
         });
     };
 
+    buildFilterRegex(searchText, includeRegistry) {
+        // Strip out all non-allowed container image characters when filtering.
+        let regexString = searchText.replace(/[^/\w_.:-]/g, "");
+        // drop registry from regex to allow filtering only by container names
+        if (!includeRegistry && regexString.includes('/')) {
+            regexString = '/' + searchText.split('/')
+                    .slice(1)
+                    .join('/');
+        }
+
+        return new RegExp(regexString, 'i');
+    }
+
     onSearchTriggered = value => {
+        let dialogError = "";
+        let dialogErrorDetail = "";
+
+        const changeDialogError = (reason) => {
+            dialogError = _("Failed to search for new images");
+            // TODO: add registry context, podman does not include it in the reply.
```

Comment:
Follow-up, or is that easy to do and you just forgot about it?

Reply:
ah, that's a moved code and that todo has probably been there for a while 🤔

Reply:
I see, nevermind then.